google-cloud-core/src/test/java/com/google/cloud/ExceptionHandlerTest.java

```diff
@@ -188,8 +186,12 @@ public RetryResult afterEval(Exception exception, RetryResult retryResult) {
         };
 
     ExceptionHandler handler = ExceptionHandler.newBuilder().addInterceptors(interceptor).build();
-    thrown.expect(NullPointerException.class);
-    handler.shouldRetry(new Exception(), null);
+    try {
+      handler.shouldRetry(new Exception(), null);
+      Assert.fail();
+    } catch (NullPointerException ex) {
+      assertNull(ex.getMessage());
+    }
   }
```

Comment:
I think here you can also use 
`@Test(expected = NullPointerException.class)`
without catching NPE and checking it's message

Reply:
done